metricbeat/module/kubernetes/kubernetes.yml

```diff
@@ -1,5 +1,26 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
+metadata:
+  name: metrics-reader
+rules:
+- nonResourceURLs: ["/metrics"]
```

Comment:
Sorry for the delay in reviewing this: Why do we need this part? I mean we can login to beat and make the curl request to metrics

Reply:
Okay, so to clarify: this file appears to be used only by tests. When we need to add new expectation files for a new Kubernetes version, we have to capture the scheduler and controller manager metrics and save them to a new file. Without this change, the system:anonymous user cannot access the metrics endpoint in order to save the data. This change is strictly required for that purpose and is unrelated to Metricbeat.